### PR TITLE
OCPBUGS-44528: Rebase etcd 3.5.17 openshift 4.18

### PR DESCRIPTION
**Pull request by Elbehery**
This PR rebases etcd 3.5.17 into openshift-4.18

running make locally 

```
SUCCESS: etcd_build (GOARCH=arm64)
./bin/etcd --version
etcd Version: 3.5.17
Git SHA: 9a0b8009e
Go Version: go1.22.9
Go OS/Arch: darwin/arm64
./bin/etcdctl version
etcdctl version: 3.5.17
API version: 3.5
./bin/etcdutl version
etcdutl version: 3.5.17
API version: 3.5
```

### CODE REVIEW

File: Dockerfile.installer [hunk 1]
```diff
@@ -38,6 +38,8 @@ FROM registry.ci.openshift.org/ocp/4.18:base-rhel9
 
 RUN yum install --setopt=tsflags=nodocs -y jq && yum clean all && rm -rf /var/cache/yum/*
 
+RUN yum install --setopt=tsflags=nodocs -y jq && yum clean all && rm -rf /var/cache/yum/*
```
Comment (tjungblu) [inline Dockerfile.installer]: that seems duplicated with the above line, is that correct?

File: Dockerfile.installer.art [hunk 2]
```diff
@@ -77,6 +77,8 @@ FROM registry.ci.openshift.org/ocp/4.18:base-rhel9
 
 RUN yum install --setopt=tsflags=nodocs -y jq && yum clean all && rm -rf /var/cache/yum/*
 
+RUN yum install --setopt=tsflags=nodocs -y jq && yum clean all && rm -rf /var/cache/yum/*
```
Comment (tjungblu) [inline Dockerfile.installer.art]: that seems duplicated with the above line, is that correct?

File: Dockerfile.installer (hunk 1 above)
Comment (Elbehery) [reply Dockerfile.installer]: fixed 👍🏽

File: Dockerfile.installer.art (hunk 2 above)
Comment (Elbehery) [reply Dockerfile.installer.art]: fixed 👍🏽

**Comment by Elbehery** [comment]
/assign @deads2k  

hello ✋🏽 

we need `valid/jira` here plz

**Comment by tjungblu** [comment]
/override ci/prow/configmap-scale

currently broken

**Comment by tjungblu** [comment]
/override ci/prow/configmap-scale

currently broken

**Pull request merged**